content/en/serverless/guide/opentelemetry.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Serverless and OpenTelemetry
+kind: documentation
+further_reading:
+  - link: '/opentelemetry/'
+    tag: 'Documentation'
+    text: 'OpenTelemetry in Datadog'
+---
+
+[OpenTelemetry][1] is an open source observability framework that provides IT teams with standardized protocols and tools for collecting and routing telemetry data.
+
+{{< img src="tracing/setup/open_standards/otel-flow.png" alt="Map options for generating telemetry data and sending it to observability products.">}}
+
+## Further Reading
+
```

Comment:
If your code is custom instrumented with the [OpenTelemetry (OTel) API](https://opentelemetry.io/docs/reference/specification/trace/api), or you want to write vendor-agnostic custom instrumentation code, you can configure it to generate Datadog-style spans and traces to be processed by the Datadog tracing library for your language, and send those to Datadog.

<add 3 headers, one for AWS, one for GCP, one for Azure>

**AWS section**

The Datadog tracing library, which is included in the Datadog Lambda Extension upon installation, accepts the spans and traces generated by OpenTelemetry-instrumented code, processes the telemetry, and sends it to Datadog. 

You can use this approach, for example, if your code has already been instrumented with the OpenTelemetry API, or if you want to instrument using vendor agnostic code with the OpenTelemetry API, while still gaining the benefits of using the Datadog tracing libraries.

To instrument AWS Lambda with OTEL API, see <go to advanced configuration docs, add a section for OTEL>. Set the environment variable DD_TRACE_OTEL_ENABLED = true 

**GCP section**

After installing <link to serverless-init tracing instructions for GCP Cloud Run>, set the following feature flag DD_TRACE_OTEL_ENABLED=true (also reference this in GCP docs )

**Azure section**

After installing <link to tracing Azure App Services or Azure Container Apps>, you can set the following feature flag DD_TRACE_OTEL_ENABLED=true (also reference this in Azure Docs)

Reply:
cc: @Danny-Driscoll feel free to add thoughts to this outline, will probably do another round of review once Cecila has made changes as well